esbuild.config.mjs

```diff
@@ -57,7 +56,6 @@ const esBuildOptions = {
   external: [
     '^react$', // Use regex ^$ because we do want to bundle 'react/jsx-transform'
     'react-dom',
-    'prop-types',
```

Comment:
We were getting console errors in SWC dev mode because an old version of prop types is being used that doesn't recognize function components.

It didn't seem to affect bundle size (might be tree-shaken out of the production bundle?), so let's bundle it. The errors go away.